BasicSample/Assets/AppRemoting/Scripts/AppRemotingSample.cs

```diff
@@ -133,15 +165,41 @@ public void ConnectToRemote(string address = null)
             StartCoroutine(Remoting.AppRemoting.Connect(remotingConfiguration));
         }
 
+        /// <summary>
+        /// Listens to the incoming connections as specified in the remotinglistenconfiguration
+        /// </summary>
+        public void ListenToRemote()
+        {
+            m_appRemotingMode = AppRemotingMode.listen;
+            StartCoroutine(Remoting.AppRemoting.Listen(remotingListenConfiguration));
+        }
+
         /// <summary>
         /// Disconnects from the remote session.
         /// </summary>
         public void DisconnectFromRemote()
         {
             Remoting.AppRemoting.Disconnect();
+            if (m_appRemotingMode == AppRemotingMode.listen)
+            {
+                m_disconnectedOnListenMode = true;
```

Comment:
I'm not sure why we need this variable.  What will happen if we don't have this?

Reply:
this can be removed

Reply:
I will actually need this because i will need to differentiate between disconnect being requested vs being in disconnected state when we are trying to listen in the beginning. This is similar reason that is explained above